library/Icingadb/Widget/Detail/MultiselectQuickActions.php

```diff
@@ -27,10 +26,45 @@ class MultiselectQuickActions extends BaseHtmlElement
 
     protected $defaultAttributes = ['class' => 'quick-actions'];
 
-    public function __construct($type, $summary)
+    /** @var ?string Url path for the {@link getLink()} method */
+    protected $urlPath;
+
+    /** @var bool Whether the rendering process-check-result button should be ignored */
+    protected $ignoreProcessCheckResultButton;
```

Comment:
please initialize this to true and change the name/semantic to `allowToProcessCheckResults`